March 2021 Newsletter (#121)
readthedocs/ethicalads.io: content/posts/newsletter-march-2021.md

Title: EthicalAds Newsletter - March 2021
Date: April 12, 2021
description: Our March 2021 ads update, including stats and commentary, on our progress as we build EthicalAds
tags: newsletter, community
authors: David Fischer
image: /images/posts/2021-mar-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@gaspanik?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Masaaki Komori</a> on <a href="https://unsplash.com/?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>

[comment]: # (The queries run to get this data, then computed in a spreadsheet for publisher rev)
[comment]: # (?start_date=2020-08-01&end_date=2020-08-31&campaign_type=All+types&revenue_share_percentage=50.0)
[comment]: # (?start_date=2020-09-01&end_date=2020-09-30&campaign_type=paid&revenue_share_percentage=70.0&sort=revenue)

## March advertising stats

March was another record month ever for advertising
although [Februrary]({filename}/posts/newsletter-february-2021.md) was better per day due to being a shorter month.

In March 2021:

> * We generated **$23,994.57** for our publishers another record but just barely up under 1% over February.

Reviewer: ```suggestion
* We generated **$23,994.57** for our publishers, another record but only 1% over February.
```

>   Of this, $10,711.54 was publishers other than our own publisher (Read the Docs).

Reviewer: ```suggestion
  Of this, $10,711.54 was to third-party publishers, with the rest to Read the Docs.
```

However, that looks nearly certain in April due to some larger advertisers shifting toward the wider ad network.

## New features

Here's the major features we added in the last month:

* On the business side, we released a [media kit]({filename}/pages/press.md)
  to help us get more traction and be part of the privacy conversation around advertising
  which is an important goal and requires constant effort.
* On our ad server, we made a number of reporting improvements for both publishers and advertisers.
  These included showing more data, giving more high level overviews of data, and speeding up our existing reports.
* We started automatically including the publisher name when redirecting to an advertiser's site.
  The advertising industry could use a bit of transparency
  and advertisers sometimes don't know where their ads are being shown.
  Our aim is to be as transparent as possible and give advertisers the ability to make more informed decisions.
* We made a small tweak to our ad display algorithm when the same person requests multiple ads within a few seconds.
  We now cache the ad and show the exact same ad and only bill advertisers a single impression for it.

>   For most publishers, this made almost no difference but for some this resulted in fewer ads being shown

Reviewer: ```suggestion
  For most publishers, this made almost no difference but for some this resulted in fewer ad views being counted
```

  and therefore less revenue.

>   We believe this is more fair to advertisers and incentivizes publishers to have good ad placements.

Reviewer: We should note this in the publisher policy as well, it's been on my list for a while, but needs to be in there.

* In our infrastructure, we've solved most of our performance/IO problems from the previous month.
  We are archiving data out of our transactional database on a regular basis after it is processed
  to help with performance.
  Longer term, we will probably use a separate reporting database for both performance
  but for now, we're keeping it simple.

> You can always see our latest updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html).

Reviewer: ```suggestion
You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html).
```

please [get in touch with any questions](mailto:ads@ethicalads.io?subject=Ads%20account%20manager%20role)
or [apply]({filename}/pages/jobs/account-manager-part-time.md)!
In addition, Read the Docs is hiring a [frontend developer](https://blog.readthedocs.com/job-frontend/).

## Upcoming features

The major features in our upcoming roadmap:

* Beginning in April, we're going to expand our advertising logic to take into account an ad's price and performance
  when choosing which ad to show.
  One advertiser has agreed to be a flex advertiser with part of their ad buys in Q2
  which will give them lower prices but not guaranteed inventory and this feature was a necessary step.
  This helps us ensure that as many ad slots have paid ads for our publishers while letting us balance our inventory.
  Initially, we expect this will drive a slight increase in revenue to publishers as we prioritize higher priced ads.
  Longer term, this aligns incentives to have better performing ads for advertisers and better performing placements on publisher sites.
* We are building improvements into both our ad client and ad server to improve our contextual targeting.

>   We believe that good contextual targeting is one of the keys to our network's success.

Reviewer: We should probably have a blog post or page on the site explaining the difference, so we can link to it from places like this. Maybe in the Ad Vision? But feels like we probably want an "explainer" which isn't a great fit there. Maybe just linked from the footer & publisher pages or something? Or maybe even just an FAQ item?

Author: I think a blog on contextual targeting would be great. Something like this: https://www.adex.network/blog/why-we-use-contextual-targeting/
